inc/updater/class-updater.php

```diff
@@ -204,12 +201,12 @@ public function repo_api_details( $result, string $action, stdClass $response )
 		}
 
 		// Exit if not our repo.
-		$slug_arr = [ $this->metadata->slug, $this->metadata->slug . '-' . get_did_hash( $this->metadata->id ) ];
+		$slug_arr = [ $this->metadata->slug, $this->metadata->slug . '-' . Packages\get_did_hash( $this->metadata->id ) ];
 		if ( ! in_array( $response->slug, $slug_arr, true ) ) {
 			return $result;
 		}
 
-		return (object) $this->get_update_data();
+		return (object) Packages\get_update_data( $this->metadata->id );
```

Comment:
If `$this->metadata->id` matches `$this->did`, should we just change all `$this->metadata->id` references to `$this->did` in this class?